.github/workflows/elixir.yml

```
@@ -68,10 +68,17 @@ jobs:
         otp: ['22', '23', '24', '25', '26']
         elixir: ['1.11', '1.12', '1.13', '1.14', '1.15', '1.16']
         exclude:
-          - {otp: '25', elixir: '1.10'}
+          - {otp: '22', elixir: '1.14'}
```

```suggestion
          - {otp: '22', elixir: '1.14'}
```

Elixir 1.14 does not officially support OTP 22 https://hexdocs.pm/elixir/1.16.2/compatibility-and-deprecations.html#between-elixir-and-erlang-otp